cli_interface.py

```
@@ -107,76 +109,78 @@ def display_progress(self, current: int, total: int, message: str = ""):
         # 获取用户输入
         answer = input("请输入你的回答：").strip()
-        
+
         # 如果有选项，检查是否输入了选项编号
         if options and answer.isdigit():
             idx = int(answer) - 1
             if 0 <= idx < len(options):
                 answer = options[idx]
-        
+
         logger.info(f"用户回答：{answer[:50]}...")
```

Ruff (0.15.9)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)